docs/data/joy/components/typography/typography.md

```diff
@@ -41,8 +41,6 @@ The `title-*` and `body-*` are commonly used in components.
 
 We recommend to combine the title and body with the same or lower size when using them together. For example, `title-lg` and `body-lg` or `title-md` and `body-sm`.
```

Comment:
```suggestion
Aside from the heading typographic levels, the Typography component provides `title-*` and `body-*` type levels.

To ensure a good UI balance and information hierarchy, we recommend combining them using the same size or a lower one.
For example, using `title-lg` with `body-lg` or `title-md` with `body-sm`.
```

I know you haven't touched this, but it's a good opportunity to revise this small paragraph a bit. I can't get the previous sentence (i.e., "The title-* and body-* are...") in the suggestion, but I'm considering it for it! It's a tricky sentence — @samuelsycamore might be able to shed some more light!